packages/slider/src/Slider.ts

```diff
@@ -160,6 +164,9 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
     @property({ type: Boolean, reflect: true })
     public override disabled = false;
 
+    @property({ type: Boolean, reflect: true, attribute: 'fill-start' })
+    public fillStart = false;
```

Comment:
I don't think this is supposed to be a boolean. I think it is supposed to represent where the fill _starts_ from. By default it would equal the same as `min` but then set it would allow for the fill to start from whereever.

Reply:
Also, maybe be unneeded work, but should we also surface a `relativeValue` property that give the value relative to the start? Calculable from the outside, so maybe not.

Reply:
should min the start value or the initial value represented as 'value' ?
fillStart boolean creates the filled highlight on the track.

Reply:
I may be misunderstand, so let's check with the design team, but I understand the workflow to be:
```
<sp-slider></sp-slider> // No fill

<sp-slider
    variant="filled"
></sp-slider> // Fill from `min`, or 0 by default

<sp-slider
    variant="filled"
    fill-start="5"
    min="0"
    max="10"
></sp-slider> // Fill from 5, or half way
```

Reply:
Sure I was wondering if fill-start="5" and value="6" then how will the slider behave? which takes precendence?

Reply:
```
0                    20
|----==0-------------|
```
I think it would look like this where the `fill-start`ed at 5 and the fill went to 6 where the handle stood. More clear might be `variant="filled" fill-start value="15" min="0" max="20"`, `variant="filled" fill-start="5" value="15" min="0" max="20"`, and `variant="filled" fill-start="5" value="0" min="0" max="20"`  as comparison:
```
0                    20
|----------====0-----|
```
```
0                    20
|----==========0-----|
```
```
0                    20
0===-----------------|
```
When `fill-start` is present, the fill starts at half the value like you've previously outlined. When `fill-start` is supplied a value, the fill starts from that value. The fill _goes_ from that number to the `value` provided. The delta from the `fill-start` to the `value` would be the `relativeValue` if we chose to surface it.

Reply:
1. For this variant `variant="filled" fill-start value="15" min="0" max="20"` , handle will be at 10 i.e the center. 
When user slides from 10 - 20 and 10 - 0 should the fill be visible for both or it should only be visible from 10-15?

2. For this variant `variant="filled" fill-start="5" value="15" min="0" max="20"` 
the handle should start from the fill-start value of "5" or from `value="15"` itself? And also do we show fill from 5-15 only or from 5-0 also?

Reply:
`variant="filled" fill-start value="15" min="0" max="20"`
In this case the "fill" will start at half (so 10) but the value will be 15, so the "fill" will take up 1/4 of the track starting from the middle.
```
0                    20
|----------====0-----|
```
---
`variant="filled" fill-start value="10" min="0" max="20"`
Fill starting at half (10) and value at 10.
```
0                    20
|----------0---------|
```
User moves handle to 20
```
0                    20
|----------==========0
```
User moves handle to 0
```
0                    20
0===========---------|
```
---
`variant="filled" fill-start="5" value="15" min="0" max="20"`
Fill takes up half of the track starting 1/4 of the way along the track.
```
0                    20
|-----=========0-----|
```
User moves to 0, fill size is 1/4 of the track starting at the beginning of the track.
```
0                    20
0=====---------------|
```

Reply:
Attached video recording for preview and review. Let me know how this solution looks now.